mtz-ajax-interceptor-behavior.html

```diff
@@ -1,13 +1,136 @@
 <link rel="import" href="../polymer/polymer.html">
 
-<!--
-`mtz.AjaxInterceptorBehavior`
+<script>
+  window.mtz = window.mtz || {};
 
-@demo demo/index.html
--->
+  /**
+   * `mtz.AjaxInterceptorBehavior` handles registering declarative interceptors then listening for all registered events
+   *  and calling the interceptors associated with those events.
+   *
+   * @polymerBehavior mtz.AjaxInterceptorBehavior
+   */
+  mtz.AjaxInterceptorBehavior = {
+    properties: {
+      /* The map of event to interceptor Set to listen against */
+      interceptors: {
+        type: Object,
+        value() {
+          return {};
+        }
+      }
+    },
+    attached() {
+      this.addEventListener('register-interceptor', this.__registerInterceptor);
```

Comment:
For attached lifecycle callback the docs specify that hybrid elements use Polymer.RenderStatus.afterNextRender to attach hybrid elements. This is snippet from docs:

For work that can be deferred until after first paint (such as adding event listeners), you can use Polymer.RenderStatus.afterNextRender, which takes the same arguments as beforeNextRender.

You've probably already seen it, but if not check out: https://www.polymer-project.org/2.0/docs/upgrade